File: Problem Solving/C/factorial.c
/*factorial of a number*/
#include<stdio.h>
#include<conio.h>
void main()
{
int fact=1,num;
printf("enter your number");
scanf("%d",&num);
for(int i=1;i<=num;i++)
{
fact=fact*i;
}
printf("%d",fact);
getch();
}

File: Problem Solving/Python/factorial.py
#factorial of number
print("Enter the number")
num=int(input())
fact=1
if(num==0):
    print("factorial is 1")
elif(num==1):
    print("factorial is 1")
for i in range(1,num+1):
    fact=fact*i
print("factorial is",fact)    
